pomodoro-client.py

```diff
@@ -74,6 +74,29 @@ def format_output(pomodoro_data, always, icon_text):
 def main():
     pass
 
+@click.command()
+@click.option('--resume-pomodoro/--no-resume-pomodoro', default=False)
+@click.option('--idle-time', default=0.5,
+              help='Minimum idle time before continuing.')
```

Comment:
shouldn't that be "maximum idle time"?

Reply:
That is actually not a bad name. I'm still horrible at naming variables

Reply:
Wait a minute. No. There is no maximum idle time, as a person could be going to the bathroom for an indefinite period of time.

Reply:
well maybe I just don't understand what's going on, because the entire issue and use case is confusing for me.

I don't know how a large enough idle time can be a valid trigger for starting a new pomodoro

Reply:
So it's basically just a buffer to fetch the time since the last were an interaction with the computer. If for example that last  action was made 0.5 seconds ago and the idle_time was set to 1 second, the new pomodoro would be started. However, I have made a better implementation and made a PR for this.